### DISCUSSION TITLE
Preview-Box like in Youtube

### DISCUSSION THREAD

**Discussion by 3flame6** [jellyfin/jellyfin-androidtv#3695] [General]
Hello,

is it possible to add a little preview box like in Youtube? Would be great.

Thanks
![Screenshot 2024-06-23 192826](https://github.com/jellyfin/jellyfin-androidtv/assets/172618979/d014bbb8-b5f8-400c-83f1-081d5fc208a8)

**Comment by nielsvanvelzen** [comment]
TrickPlay support is planned
